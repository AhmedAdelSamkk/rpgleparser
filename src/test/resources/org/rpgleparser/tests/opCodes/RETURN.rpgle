     C                   RETURN(H) MYVAR + 10
          return(h) (myvar + 10);