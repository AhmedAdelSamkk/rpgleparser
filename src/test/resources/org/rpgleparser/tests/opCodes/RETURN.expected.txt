/*===TOKENS===*/
CS_FIXED                 <C>
BlankIndicator           <  >
BlankFlag                < >
BlankIndicator           <  >
CS_BlankFactor           <              >
OP_RETURN                <RETURN>
'('                      <(>
CS_OperationAndExtender  <H>
')'                      <)>
ID                       <MYVAR>
'+'                      <+>
NUMBER                   <10>
C_FREE_NEWLINE           <
>
OP_RETURN                <return>
'('                      <(>
CS_OperationAndExtender  <h>
')'                      <)>
'('                      <(>
ID                       <myvar>
'+'                      <+>
NUMBER                   <10>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (cspec_fixed
      C
      (cs_controlLevel '  ')
      (onOffIndicatorsFlag ' ')
      (cs_indicators '  ')
      (factor '              ')
      (cspec_fixed_standard
        (csRETURN
          RETURN
          (cs_operationExtender ( H ))
          (c_free
            (expression
              (expression (identifier (free_identifier MYVAR)))
              +
              (expression (number 10))
            )
          )
          \r\n
        )
      )
    )
  )
  (statement
    (free
      (baseExpression
        (op
          (op_return
            return
            (cs_operationExtender ( h ))
            (expression
              (
              (expression
                (expression (identifier (free_identifier myvar)))
                +
                (expression (number 10))
              )
              )
            )
          )
        )
      )
      ;
    )
  )
)
/*======*/